app/models/policy_event/purging.rb

```diff
@@ -12,16 +12,6 @@ def purge_window_size
         ::Settings.policy_events.history.purge_window_size
       end
 
-      def purge_queue
-        MiqQueue.put_unless_exists(
-          :class_name  => name,
-          :method_name => "purge",
-          :role        => "event",
-          :queue_name  => "ems"
-        )
-      end
-      alias_method :purge_timer, :purge_queue
-
```

Comment:
thanks for keeping me honest.

Yes, most of these come from the default implementation